config/testgrids/kubernetes/sig-storage/config.yaml

```diff
@@ -82,6 +82,12 @@ dashboards:
       description: storage snapshot e2e tests for master branch
       alert_options:
         alert_mail_to_addresses: kubernetes-sig-storage-test-failures@googlegroups.com
+    - name: kind-disruptive
```

Comment:
BTW you can use an annotation on the job to avoid having to configure this file if you move the whole dashboard to those IIRC. I'm not sure if this works with mixing annotations and on-disk dashboard. It's by design that _blocking_ presubmits require approval but not optional ones like this.

you set `testgrid-dashboards` and `testgrid-tab-name` as an `annotation` on the job.
https://github.com/kubernetes/test-infra/blob/be868a8514747bf370a38f33ace1518c5fc99982/testgrid/config.md#prow-job-configuration

Reply:
ok, using annotations now